frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionDetails.tsx

```diff
@@ -60,13 +78,24 @@ function ModelActionDetails({
           {t`Enable implicit actions`}
         </Button>
       )}
-      <AddActionButton modelId={modelId} />
+      <Button
+        onClick={toggleIsActionCreatorVisible}
+        icon="add"
+      >{t`Create a new action`}</Button>
       <ul>
         {actions.map(action => (
           <li key={action.id}>
             <ActionListItem
-              to={`/action/${action.id}`}
+              borderless
               disabled={isImplicitAction(action)}
+              onClick={
+                !isImplicitAction(action)
+                  ? () => {
+                      setEditingActionId(action.id);
+                      toggleIsActionCreatorVisible();
+                    }
+                  : undefined
+              }
```

Comment:
I personally find this `condition ? () => {} : undefined` kind of expressions a bit hard to read sometimes. Should we maybe always pass a function, but do the check inside? Like:

```jsx
onClick={() => {
  if (isExplicitAction(action)) {
    setEditingActionId(action.id);
    toggleIsActionCreatorVisible();
  }
}}
```